tables/fr-bfu-comp6.utb

```diff
@@ -1,5 +1,21 @@
 # liblouis: Unified French 6 dots Braille table
 
+# -----------
+#+locale:fr
+#+type:computer
+#+dots:6
+#+bfu
```

Comment:
What is `bfu`?

Reply:
Braille Français Unifié. It's just a way to differentiate with other French tables. If we think this table should be the default for French computer braille, then there is a problem with the metadata.

Reply:
But shouldn't this be some form of metadata just like ueb? Ah I see that ueb is handled just like that. But then shouldn't meta data always be `key: value`?

Reply:
Both are supported.

Reply:
See http://liblouis.org/documentation/liblouis.html#Table-Metadata

Reply:
Oh, I didn't know that. I'd only support `key: value`. a) to be orthogonal and b) to make the syntax easier on the user

Reply:
You are a bit late with that feedback :) I liked it. But can we leave that discussion for another time?